server/docker/Dockerfile

```diff
@@ -55,48 +61,16 @@ RUN set -eux; \
     	  ; \
     rm -f /tmp/openjdk.tar.gz /usr/local/java/lib/src.zip;
 
-########################################################################################################################
-#
-# Build Final Image
-#
-########################################################################################################################
-FROM ubuntu:${UBUNTU_TAG} AS openjdk-base
-
-# Define Standard Environment Variables
-ENV LC_ALL=C.UTF-8
-ENV DEBIAN_FRONTEND=noninteractive
-ENV JAVA_VERSION="jdk-21.0.1+12"
-ENV JAVA_HOME="/usr/local/java/"
+########################################
+####    Deterministic Build Hack    ####
+########################################
 
```

Comment:
These lines seem out of place and should be above the actual workaround starting at line 94.